plugins/jenkins/test/models/samson/jenkins_test.rb

```diff
@@ -19,7 +19,7 @@ def stub_job_detail
   def stub_build_with_parameters
     stub_request(:post, "http://www.test-url.com/job/test_job/buildWithParameters").
       with(
-        body: {"buildStartedBy" => "Super Admin", "originatedFrom" => "Project_Staging_staging", "commit" => "staging"},
+        body: {"buildStartedBy" => "Super Admin", "originatedFrom" => "Project_Staging_staging", "commit" => "staging", "deployUrl" => "http://www.test-url.com/projects/foo/deploys/178003093"},
```

Comment:
might be better to use `#{deploy.id}` or `#{Deploy.first.id}` or something like that here ... that id might change randomly